ngraph/frontend/onnx_import/src/op/softmax.cpp

```diff
@@ -40,15 +40,9 @@ namespace ngraph
                     std::make_shared<default_opset::Subtract>(coerced_data, max);
 
                 const auto result = std::make_shared<default_opset::Softmax>(data_minus_max, 1);
-                if (data.get_partial_shape().is_static())
-                {
-                    return ngraph::builder::opset1::reshape(result, data.get_shape());
-                }
-                else
-                {
-                    const auto data_shape = std::make_shared<default_opset::ShapeOf>(data);
-                    return std::make_shared<default_opset::Reshape>(result, data_shape, false);
-                }
+                const auto data_shape = std::make_shared<default_opset::ShapeOf>(data);
+                const bool special_zero = false;
+                return std::make_shared<default_opset::Reshape>(result, data_shape, special_zero);
```

Comment:
These changes should not belong to your review. Please, merge master branch properly.